notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    tot_flops
</pre><p>https://www.hyro.ai/glossary/gpt-3 claims GPT-3 took 3.14e23 FLOPS, so the full KataGo training run is only a factor of 4 fewer FLOPS!</p><p> </p><p>Probably a lot cheaper in $ terms though because it can use GPUs with smaller VRAM and without high-speed interconnects between them though.</p><p> </p><p>As another sanity check, 8e22 FLOPS is ~3000 A100 days taking the 312 TFLOPS figure for the A100 at face value. That seems like right OOM at least.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>